docs/workers.md

```diff
@@ -412,3 +412,9 @@ It can be launched from command-line with:
 ```sh
 $ cozy-stack jobs run migrations --domain example.mycozy.cloud --json '{"type": "to-swift-v3"}'
 ```
+
+## index
+
+This worker is used for sending data to a RAG. It looks at the changes feed for
+the given doctype, send the changes to an external indexer that will generate
+embeddings for the data and put them in a vector database.
```

Comment:
Since we're using the generic terms `index` and `indexer` shouldn't we stick to that here as well?
This would give something like:

This worker looks for changes to the given doctype in CouchDB's changes feed and send the associated documents to an external indexer.
The index is stored in the indexer's own database.
For `io.cozy.files` documents, the file content itself is sent to the indexer as well.